examples/classification.ipynb

```diff
@@ -5,7 +5,11 @@
    "id": "cb9e5d62",
```

Comment:
<p>Maybe remove the second word <em>methods </em>in the&nbsp;second paragraph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>